initfiles/componentfiles/configxml/mysqlserver.xsd

```diff
@@ -116,19 +116,25 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="computer" type="computerType" use="required">
+            <xs:attribute name="computer" type="computerType" use="optional">
                 <xs:annotation>
                     <xs:appinfo>
                         <tooltip>The computer on which the MySQL database is deployed.</tooltip>
                         <autogenforwizard>1</autogenforwizard>
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
+            <xs:attribute name="host" type="xs:string" use="optional">
+                <xs:annotation>
+                    <xs:appinfo>
+                        <tooltip>The host name on which the MySQL database is deployed. This is used before computer option.</tooltip>
```

Comment:
There are several use cases we have encountered.

1. The computer way always require an IP address, and the ConfigMgr always use the IP in the output configure XML. In case we have a host name, and its ip changes, hard code IP in the config is not good. We'd like to have the host name directly in the config.

2. There are VM setup, where localhost/127.0.0.1 works, but the real IP does not work. The ConfigMgr always output the real IP in the config. This was original reason we added host name during the China JV training.